media/editor/dataInspectorProperties.tsx

```diff
@@ -1,3 +1,49 @@
+/** Reads a ULEB128 at offset 0 from the buffer. */
+const getULEB128 = (arrayBuffer: ArrayBuffer) => {
+	const buf = new Uint8Array(arrayBuffer);
+
+	let result = 0n;
+	let shift = 0n;
+	let index = 0;
+	try {
+		while (true) {
+			const byte: bigint = BigInt(buf[index++]);
+			result |= (byte & 0x7Fn) << shift;
+			if ((0x80n & byte) === 0n) {
```

Comment:
sorry, I didn't realize in your initial issue that this was a variable length value.

I think we should have some limits here otherwise a giant file of `0xFF`'s could dramatically slow down the editor.

Reply:
Yes, sure!